### DISCUSSION TITLE
Asus AI Noise Canceling Microphone

### DISCUSSION THREAD

**Discussion by nilantonijoan** [seerge/g-helper#1317] [Ideas]
Hey Seerge, first of all, I'm thankful for your work and what you've achieved. 

The only thing I miss from the official software is the noise canceling for the microphone.

(At least for me worked very well)

I'm begging you if it's possible to add it to ghelper.

Thanks

PD: when talking with friends in discord they hear my voice as if I were a 60 yo packing 50 cigarettes a day.

**Comment by seerge** [comment]
@nilantonijoan hello, there is a separate topic for that https://github.com/seerge/g-helper/issues/18
It all depends on your model, new devices can set same settings via Realtek console app

**Reply by nilantonijoan** [reply]
So in the realtek console of my laptop (M16 2023) I can set the same ai noise cancelation as Armoury Crate?

Sorry for wrongly flaring this post

**Reply by seerge** [reply]
Yes, I believe (and other people also say that), that AC just sets realtek settings. Try to experiment with them / and audio settings of your mic in windows (i.e. not apply any custom effects there)

**Reply by nilantonijoan** [reply]
Perfect, thanks a lot.

**Reply by IceStormNG** [reply]
> So in the realtek console of my laptop (M16 2023) I can set the same ai noise cancelation as Armoury Crate?

Yes and no. There are 2 microphone modes, the one from ForteMedia, which can be set in the Realtek app (In AC they're often labeled "High Quality", they're the modes on the left side).
The AI Modes (right side) cannot be used. Those run through a virtual audio device and a service from Armoury Crate. If you want that, you could also use a 3rd party service like Nvidia broadcast.
